disorder-core/src/Disorder/QuickCheck.hs

```diff
@@ -0,0 +1,6 @@
+module Disorder.QuickCheck (
```

Comment:
> yeah I wouldn't export it as part of Disorder.Core - as that changes the contract with people already using Disorder.Core and Test.QuickCheck (they would get redundant import warnings all of a sudden, which is a tad extreme).

I agree, but i think the module should still be `Disorder.Core.QuickCheck` (and doesnt need to be re-exported from `Disorder.Core`.